ports/nrf/boards/raytac_mdbt50q-db-40/mpconfigboard.mk

```diff
@@ -0,0 +1,8 @@
+USB_VID = 0x239A
+USB_PID = 0x802A
```

Comment:
Please don't reuse the PID. See the learn guide for requesting a new PID.

Reply:
Request submitted.

Reply:
PID changed.